library.properties

```
@@ -1,4 +1,4 @@
-name=Modulino
+name=Arduino Modulino
```

```suggestion
name=Arduino_Modulino
```

The policy is that the names of all official Arduino libraries have the prefix `Arduino_`:

https://forum.arduino.cc/t/new-official-policy-for-the-use-of-arduino-in-library-names/580564/4

> Our official libraries from now on we'll be called Arduino_Something

https://arduino.github.io/arduino-lint/dev/rules/library/#name-missing-official-prefix-lp013

> ### name missing official prefix (`LP013`)

>
>The `name` field in the library's `library.properties` metadata file is missing the "Arduino_" prefix. The names of all new official libraries must have this prefix.